helpers/dateFormatter.js

```diff
@@ -0,0 +1,47 @@
+
+
+const DateFormatter = (props) => {
```

Comment:
A small change you need to do. It is a helper function. not a render function.  so function name should be same as file name. and function is called using function name like dateFormatter('date to be formatter')